apps/quickbooks_online/utils.py

```
@@ -656,19 +659,23 @@
-        for i in range(0, len(active_existing_classes), BATCH_SIZE):
-            batch = active_existing_classes[i: i + BATCH_SIZE]
-            with transaction.atomic():
-                DestinationAttribute.objects.filter(
-                    attribute_type='CLASS',
-                    workspace_id=self.workspace_id,
-                    destination_id__in=batch,
-                    active=True
-                ).update(active=False, updated_at=timezone.now())
+        inactive_classes = self.connection.classes.get_inactive(sync_after)
+
+        for inactive_classes in inactive_classes:
```

Ruff (0.12.2)

Loop control variable `inactive_classes` overrides iterable it iterates

(B020)